tests/integration/dual-channel-replication.tcl

```diff
@@ -23,6 +23,12 @@ proc get_client_id_by_last_cmd {r cmd} {
     return $client_id
 }
 
+proc wait_and_resume_process idx {
+    set pid [srv $idx pid]
+    wait_for_log_messages $idx {"*Process is about to stop.*"} 0 2000 1
+    exec kill -SIGCONT $pid
```

Comment:
for the sake of modularity, can we call resume_process gere instead (I know it is basically doing just that)